### ISSUE TITLE
TS Variable types wrong for operations with default variables

### ISSUE THREAD

**Issue by NickClark** [dotansimha/graphql-code-generator#1564]
**Describe the bug**
The types generated for operations with variables that have set defaults are wrong, and force a value. 
**To Reproduce**

1. My GraphQL schema:

```graphql
extend type Mutation {
  setUserActive(id: ID!, active: Boolean!): Boolean
}
```

2. My GraphQL operations:

```graphql
mutation SetUserActive($id: ID!, $active: Boolean = true) {
  setUserActive(id: $id, active: $active)
}
```

3. My `codegen.yml` config file:

```yml
overwrite: true
documents:
  - src/client/**/documents.graphql
schema:
  - src/client/**/*schema.graphql
generates:
  src/client/generated/graphql.ts:
    config:
      avoidOptionals: false
      immutableTypes: true
    plugins:
      - typescript
      - typescript-operations
      - typescript-apollo-angular
  src/client/generated/schema-client.graphql:
    plugins:
      - "schema-ast"
```

**Expected behavior**

I expect the generated Variable type to allow `active` to be optional

**Environment:**

- OS: 10.13.6
- `@graphql-codegen/..@1.0.4`: 
- NodeJS: 10

**Comment by kamilkisiela** [comment]
#1567 Should fix it

**Comment by kamilkisiela** [comment]
I released a canary version, could you try: `1.0.5-alpha-4b26c306.0+4b26c306`?

**Comment by dotansimha** [comment]
Fixed in 1.0.5.
